docs/modules/databases/jdbc.md

```diff
@@ -55,6 +55,11 @@ Insert `tc:` after `jdbc:` as follows. Note that the hostname, port and database
 
 `jdbc:tc:tidb:v6.1.0:///databasename`
 
+#### Using YugabyteDB
+
+`jdbc:tc:yugabytedb:2.14.4.0-b26:///databasename`
```

Comment:
We can keep `yugabyte`. We can revisit on the next major version.

Reply:
@srinivasa-vasu can you update the PR taking into account the suggestions, please?

Reply:
@eddumelendez - Apologies, my bad, totally missed it. Reverted just to the doc update. Can we update the `tc url` to `yugabytedb` in the next major release?